notebooks/pass_request_uri.ipynb

```diff
@@ -0,0 +1,73 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
```

Comment:
Was it intended to commit the Notebook? If yes, would you mind cleaning the outputs? :)

Reply:
Notebook was just to showcase it as a POC. Not intended to be merged in. I'll remove it :)